src/azure-cli/azure/cli/command_modules/network/aaz/latest/network/application_gateway/settings/_update.py

```diff
@@ -16,15 +16,12 @@
 )
 class Update(AAZCommand):
     """Update settings.
-
-    :example: Update settings to use a new probe.
-        az network application-gateway settings update -g MyResourceGroup --gateway-name MyAppGateway -n MySettings --probe MyNewProbe
```

Comment:
thanks @william051200's investigation, the empty examples come from version 2024 and inheritance from 2023 works well.